.github/workflows/build-runner.yml

```diff
@@ -19,16 +31,21 @@ jobs:
         with:
           buildx-version: latest
 
-      - name: Login to GitHub Docker Registry
-        run: echo "${DOCKERHUB_PASSWORD}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
-        env:
-          DOCKERHUB_USERNAME: summerwind
-          DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_ACCESS_TOKEN }}
-
       - name: Build Container Image
         working-directory: runner
         run: |
           docker buildx build \
+            --build-arg RUNNER_VERSION=${RUNNER_VERSION} \
+            --build-arg DOCKER_VERSION=${DOCKER_VERSION} \
             --platform linux/amd64,linux/arm64 \
-            --tag summerwind/actions-runner:latest \
-            -f Dockerfile . --push
+            --tag summerwind/actions-runner:v${RUNNER_VERSION} \
+            -f Dockerfile .
```

Comment:
Now you're no longer using `--push` you'll need to provide an alternative. To be honest splitting this out will make it much more difficult because fat manifests won't load into docker the same way single platform architectures do.